### DISCUSSION TITLE
各位大师兄，我想问一下，web PC端可以拖拽里面的图片或者文字吗？

### DISCUSSION THREAD

**Discussion by hubeishuaige** [Tencent/libpag#1954] [Q&A] [answered]
各位大师兄，我想问一下，web PC端可以拖拽里面的图片或者文字吗？我想实现点击哪个图片或者文字，就可以实现选中状态，并且可以放大或者缩小，或者拖拽吗？比如说我选中文字这个图层，可以标记选中区域，类似于下面这种图片效果：
![pag-demo](https://github.com/Tencent/libpag/assets/21007762/fa2a11a6-bf2c-49e1-b5a1-cb75adba2d64)

**Comment by kevingpqi123** [comment, marked as answer]
提供一种实现思路：
PAGView 提供了 getLayersUnderPoint 的接口用于获取某个点击点下面的所有图层，通过 getBounds 的接口可以获取到相对于 PAGView 的位置信息，然后通过原生方法自己添加一个选择框，缩放的时候，PAGLayer 有一个 setMatrix 的接口支持缩放、位置和扭曲

**Reply by leohandsome1688** [reply]
大佬。可以告知一下怎么利用getBounds 方法获取到的位置信息，转成在canvas上面的区域大小或者位置信息吗？非常感谢~
